### PR TITLE
data: Saha-7.json

### PR DESCRIPTION
**Pull request by Saha-7**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7665"></a>

### CODE REVIEW

File: data/Saha-7.json [hunk 1]
```diff
@@ -1,16 +1,104 @@
...
+    },
+    {
+      "title": "NAPPTIVE Hackathon",
+      "date": "April 2023",
+      "icon": "FaLink",
+      "color": "green",
+      "description": "Successfully completed the hackathon with the team Organized by NAPPTIVE x WeMakeDevs",
+      "url": "https://wemakedevs.org/events/hackathons/napptive",
+      "url": "https://poonam1607.hashnode.dev/oamazingly-naptivating-unleashing-the-power-of-oam-on-the-napptive-platform",
+      "url": "https://playground.napptive.dev/app/default/metabase-app"
```
Comment (SaraJaoude) [inline data/Saha-7.json:101]: JSON does not allow for multiple properties, in this case URLs. 
Please pick one URL and delete the other two.

Comment (Saha-7) [reply data/Saha-7.json]: Okay done.

**Review by SaraJaoude — APPROVED** [review]
looks good now

**Comment by Saha-7** [comment]
Thanks 🤗

**Pull request merged**